bundles/asm-policy-v0.0.1/README.md

```diff
@@ -0,0 +1,53 @@
+# asm-policy-v0.0.1
+
+## Description
+Anthos Service Mesh security policy constraints bundle.
+
+## Setters
+
+-   `enforcementAction`: The enforcement mode of the constraints, options are `dryrun` (default) or
+    `deny`.
+-   `root-namespace`: The root namespace of the Anthos Service Mesh, default as `istio-system`.
```

Comment:
We can leave this for now, but I wonder if it should be picked up by the set-namespace function. @yuwenma Thoughts?

Reply:
[set-namespace](https://catalog.kpt.dev/set-namespace/v0.2/) seems not align with the use case here.

Reply:
@morgante yes, the new set-namespace will better handle this. @liwenhao0810 we are working on a new version and the kpt.dev will be updated soon.

Reply:
After reading, it designs more for a namespaced resource, while this is not really the case though. This is more a param that is used by the API, the meaning of this param is the ASM root namespace value. This value shouldn't be set to any other namespace by any cases.

Reply:
If this value should nearly always be `istio-system`, let's just remove the setter then.

Reply:
This value is `istio-system` by default, in current ASM setup user can possibly setup the mesh with different root namespace.

Reply:
How often is that different? Remember, users can always edit values in-place. We don't need a setter for everything users *might* change.

Reply:
That's an interesting point, I thought the reason having these setters is to help users in scripted use case like CI/CD, otherwise it always be easier to edit in place.
I don't see the concern having this setter, unless I'm missing something.

Reply:
In general we want to avoid overusing setters. They're useful when you have a single value which shows up many places and you want to set it once for all but beyond that they're usually introducing an unnecessary layer of indirection.

Reply:
Sounds good, after double checking with ASM team, feel comfortable to remove this setter, please check.